qreader.py

```diff
@@ -227,7 +225,7 @@ def detect_and_decode(
         if return_detections:
             return decoded_qrs, detections
         else:
-            return decoded_qrs
+            return decoded_qrs, None
```

Comment:
It should still returning only one parameter instead of two when return detections is False to avoid including breaking changes

Reply:
Oh good catch!!

Reply:
Will amend in another PR – i think this change got introduced to make the mypy "easier" to manage